src/Database/Version460Update.php

```diff
@@ -177,4 +180,111 @@ public function run(): void
 
         $this->connection->query('UPDATE tl_content SET playerStop = youtubeStop');
     }
+
+	/**
+	 * Replacement for runonce.php migration of ce-access extension (terminal42/contao-ce-access) which has been merged
+	 * into contao/core-bundle in version 4.6.0.
+	 */
+	private function migrateCeAccess(): void
+	{
+		// v1 -> v2
+		$contentElements = array();
+
+		foreach ($GLOBALS['TL_CTE'] as $k => $v) {
+			$contentElements[$k] = array();
+			foreach ($v as $kk => $vv) {
+				$contentElements[$k][] = $kk;
+			}
+		}
+
+		$this->ceAccessInvertElements('tl_user', $contentElements);
+		$this->ceAccessInvertElements('tl_user_group', $contentElements);
+
+		$modules = array();
+
+		// v2 -> v3
+		foreach ($GLOBALS['BE_MOD'] as $moduleConfigs) {
+			foreach ($moduleConfigs as $moduleName => $moduleConfig) {
+				// Skip modules without tl_content table
+				if (!\in_array('tl_content', (array) $moduleConfig['tables'])) {
+					continue;
+				}
+
+				$modules[] = $moduleName;
+			}
+		}
+
+		$this->ceAccessGroupElements('tl_user', $modules);
+		$this->ceAccessGroupElements('tl_user_group', $modules);
+	}
+
+	/**
+	 * Convert negative-selection of column 'contentelements' in tl_user_group and tl_user to additive selection in the
+	 * column 'elements'
+	 *
+	 * @param string $table
+	 * @param array  $contentElements
+	 */
+	private function ceAccessInvertElements(string $table, array $contentElements): void
+	{
+		$columns = $this->connection->getSchemaManager()->listTableColumns($table);
+
+		if (!isset($columns['contentelements']) || isset($columns['elements'])) {
+			return;
+		}
+
+		// Add the new field to the database table
+		$this->connection
+			->query("ALTER TABLE $table ADD COLUMN elements blob NULL");
+
+		$records = $this->connection
+			->query("SELECT id, contentelements FROM $table WHERE contentelements!=''")
```

Comment:
What if the field is `NULL` and not an empty string?

Reply:
Has this column been nullable? If so we should probably account for that. It would have crashed in the extension then as well: https://github.com/terminal42/contao-ce-access/blob/dd35f7d67bfa72266ae024a4f28fff235809b547/config/runonce.php#L49-L52

Reply:
...ahh yes, seems so. https://github.com/terminal42/contao-ce-access/blob/c8b6b71e30b8e9ae89e7813176408d75fd7abf0d/config/database.sql#L15-L28

Will change this to handle `NULL` as well.